inst/htmlwidgets/leaflet.js

```diff
@@ -856,6 +858,36 @@ var dataframe = (function() {
     });
   };
 
+		// TO DO: right now using .id but may let users choose, let users choose "<" or ">" or "=" (default)
+		// consider letting users specify listed style arguments ala addGeoJSON
+		// and create methods for many ids to style or remove with single loop, and many ids to many styles
```

Comment:
Yes, it will be nice to vectorize over id or/and style.

(nitpicking: could you replace tabs with spaces here?)